std/typecons.d

```diff
@@ -64,67 +69,44 @@ static if (is(T:Object))
 else
     alias RefT = T*;
```

Comment:
I think we'll figure out a solution for classes and nested classes as well. Let's take them out of the calculation for now.